app/javascript/components/Home/Service/WorthPhotoContainer.1.jsx

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from "react";
+import { FadeInLeft } from "../../Constants/Constants";
+import {
+  postFetch,
+  deleteFetch,
+  getFetch
+} from "../../Constants/FetchComponent";
+import { DeleteButton, AddButton } from "../../Constants/Buttons";
+
+class WorthPhotoContainer extends Component {
```

Comment:
Hey Tony, so the whole component can actually be written using the new React Hooks functionality. Check this out to get an idea of how newer React components are being written now. https://reactjs.org/docs/hooks-intro.html

Reply:
Not Necessary but could be good practice

Reply:
correct, i think my next project will be hooks instead of massively overhauling this repo.

Reply:
It actually wouldn't require a massive overhaul and would be some solid practice! You already have a recent enough version of React in order to use it, you would just have to import the `useEffect` and `useState` hooks when needed from the library!

Reply:
you're right, it's not that bad it looks like for this component. I just meant i'd have to change the other 30+ ones. But it is good practice!